doc/schemas/options_schema.json

```diff
@@ -301,6 +313,7 @@
     "Terrains",
     "Cliques",
     "AlphaCalculatorMode",
-    "CombinerModuleGroup"
+    "CombinerModuleGroup",
+    "DebugTileResolution"
```

Comment:
Hmm... probably not. I'll see if I can get spotify-json to use a sensible default. I've been using `(256,256) as the default debug tile size for `(32,32)` tiles; maybe `tile size * (8,8)` would be a good default.